src/toil/batchSystems/kubernetes.py

```diff
@@ -797,54 +890,150 @@ def mount_host_path(volume_name: str, host_path: str, mount_path: str, create: b
 
         return pod_spec
 
+    def _release_acquired_resources(self, resources: List[int], notify: bool = False) -> None:
+        """
+        Release all resources acquired for a job.
+
+        :param resources: The resources in the order: core, memory, disk.
+        :param notify: If True, notify the threads that are waiting on the
+                       self._work_available condition.
+        """
+        # TODO: accelerators?
+
+        # What pools and sets do we want resources from
+
```

Comment:
This doesn't really make sense to me.
```suggestion
```

Reply:
Yeah, this was also from the single machine batch system where we have ResourceSets for the accelerators. (We might want to consider the accelerators here as well, but I wanted to make sure the resource pool works first).